tasks/task_14/client/handler/handler.go

```diff
@@ -0,0 +1,76 @@
+package handler
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"time"
+
+	"studing/tasks/task_14/server/models"
+)
+
+type Handler struct {
+	conn    net.Conn
+	scanner scanner
+}
+
+func NewSender(conn net.Conn, s scanner) *Handler {
+	return &Handler{
+		conn:    conn,
+		scanner: s,
+	}
+}
+
+type scanner interface {
+	Scan(ctx context.Context) (string, error)
+}
+
+func (h *Handler) Validate(ctx context.Context) error {
+	str, err := h.scanner.Scan(ctx)
+	if err != nil {
+		return fmt.Errorf("error while scanning: %w", err)
+	}
+
+	requestMessage := &models.RequestMessage{
+		Text: str,
+	}
+
+	message, err := json.Marshal(requestMessage)
+	if err != nil {
+		return fmt.Errorf("error while marshalling request: %w", err)
+	}
```

Comment:
please create method `ToBytes` on structure, to avoid duplication of transforming structure to bytes